### PR TITLE
Change app into pip package

### PR DESCRIPTION
**Pull request by silverhairs**
### What has changed:
1. There is no longer install.sh or run.py. The app now acts like a full pip package (It is one actually)
2. The user no longer needs to input their linkedin credentials in a file, the cli will request them.
3. The **lib** directory has been renamed to **core**﻿

### What should be done by now:
1. Writing tests
2. Hosting the package on PyPI and make it accessible to the entire world

### CODE REVIEW

File: llvd/__init__.py [hunk 1]
```diff
@@ -20,47 +24,44 @@
...
+    link = config.main_url + str(course).replace(" ", "-").replace(":-", "-").replace(
+        "-&", ""
+    ).replace(".", "-")
+    email = click.prompt("Please enter your linkedin email address: ")
+    password = click.prompt("Enter your Linkedin Password: ", hide_input=True)
     try:
         browser = create_browser()
     except WebDriverException:
-        print("chromedriver is missing")
+        click.echo(f"{BOLD}Please install chromedriver", err=True)
```
Comment (knowbee) [inline llvd/__init__.py:63]: The argument err=True throws a TypeError..you might want to remove it.

Comment (knowbee) [inline llvd/__init__.py:63]: The argument err=True throws a TypeError..you might want to remove it.

File: llvd/__init__.py [hunk 2]
```diff
@@ -20,47 +24,44 @@
...
 
-    link = config.main_url + str(course).replace(" ", "-").replace(":-",
-                                                                   "-").replace("-&", "").replace(".", "-")
-    if(config.email == "" or config.password == ""):
-        print("missing credentials llvd --help")
-        sys.exit(0)
+    link = config.main_url + str(course).replace(" ", "-").replace(":-", "-").replace(
+        "-&", ""
+    ).replace(".", "-")
+    email = click.prompt("Please enter your linkedin email address: ")
```
Comment (knowbee) [inline llvd/__init__.py:58]: colon is not required, click.prompt has it by default

**Review by knowbee — CHANGES_REQUESTED** [review]

File: llvd/__init__.py (hunk 1 above)
Comment (knowbee) [reply llvd/__init__.py:63]: run `llvd` without arguments to reproduce the error

Comment (silverhairs) [reply llvd/__init__.py:63]: That's very weird, [from click 3.0 to above, that error is not supposed to appear](https://click.palletsprojects.com/en/7.x/utils/).
Anyway, let me remove it.

**Review by knowbee — APPROVED** [review]

**Pull request merged**